src/component/organism/list/MyPetList.js

```diff
@@ -3,8 +3,7 @@ import {FlatList, TouchableOpacity} from 'react-native';
 import {Text, View} from 'react-native';
 import AddPet from 'Molecules/button/AddPet';
 import PetImageLabel from 'Molecules/label/PetImageLabel';
-import {myPetList} from 'Organism/style_organism';
-
+import {myPetList} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {myPetList} from 'Organism/style_organism copy';
```